build.gradle.kts

```diff
@@ -5,10 +5,9 @@ import org.jetbrains.kotlin.config.KotlinCompilerVersion.VERSION as KOTLIN_VERSI
 
 @Suppress("DSL_SCOPE_VIOLATION")
 plugins {
-//    signing
```

Comment:
This plugin is added in our `local.figure.publishing` composite plugin - this can be found in `./build-logic`